.gitignore

```diff
@@ -3,3 +3,6 @@
 
 # Ada Library Information
 *.ali
```

Comment:
Please, kindly delete lines 1 to 5 in this file. They are not needed here. Deleting them ensures the code is clean 😄